### DISCUSSION TITLE
Disable arrow left and right in zoom plugin?

### DISCUSSION THREAD

**Discussion by carestad** [igordanchenko/yet-another-react-lightbox#246] [Q&A] [answered]
I am looking for an option to customise (or disable completely) right/left arrow keys when using the zoom plugin. Is there any way to achieve that?

**Comment by igordanchenko** [comment]
Hi there! Can you describe the end result you want to achieve? Right/left arrow keys are used to pan zoomed-in image (move zoomed-in image to the left / to the right)

**Reply by carestad** [reply]
I would like to keep the carousel next/previous handling that is the default and when an image hasn't been zoomed :sweat_smile:  Is there any way to achieve that you think?

**Reply by igordanchenko** [reply]
Isn't that how it works out of the box? When an image isn't zoomed in, the arrow keys control navigation to the previous/next slide. When an image is zoomed in, the arrow keys control the panning of the image.

**Reply by igordanchenko** [reply]
Or are you saying you want to keep carousel previous/next navigation handling even when an image is zoomed in?

**Reply by carestad** [reply]
> Or are you saying you want to keep carousel previous/next navigation handling even when an image is zoomed in?

Yes, this is what I want to achieve ☺️

**Reply by igordanchenko** [reply, marked as answer]
No, I'm afraid this is not supported.
